src/templates/admin/elements/metadata.html

```diff
@@ -1,6 +1,9 @@
 {% load settings %}
+{% load roles %}
 
-{% if not view_type == "author" %}
+{% user_has_role request 'editor' as user_is_editor %}
+
+{% if not view_type == "author" and user_is_editor %}
```

Comment:
I'm a bit confused here, from the original issue it looks like you agreed with David that the typesetter should have access to this view and to add notes(below) but the code restricts them from doing so

Reply:
Yes the wording was confusing but the outcome is what I always intended except instead of making a special model for typesetters I simply hid the bits they can’t use...

Reply:
For avoidance of doubt: typesetters should not be allowed to edit metadata or add notes to articles.

Reply:
Ok cool, the view in the backend is still decorated with `@production_user_or_editor_required` which seems to contradict this logic.

Reply:
Good catch, fix committed.